index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+
+
+<html lang="en-GB">
+
+<head>
+	<title>	
+		Paul English
+	</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" rel="stylesheet" type="text/css">
+	<link href="styles.css" rel="stylesheet" type="text/css">	
+</head>
+
+
+
+<body>
+	<div class="container">	
+		
+		
+		<nav>
+			<div>
+				<div class="navigation navBorder">
+					<a class="x" href="#AboutMe">About Me </a>
+				</div>
+				<div class="navigation navBorder">
+					<a href="#Portfolio">Portfolio </a>
+				</div>
+				<div class="navigation navBorder">
+					<a href="#ContactMe">Contact Me </a>
+				</div>
+			</div>
+		</nav>
+	
+		<header>
+			<div>	
+				<h1> Paul English (alias Zedonk or Zedonk1995) </h1>	
+			</div>
+		</header>
+		
+		<div class="titleBackgroundFooter">
+			<footer>
```

Comment:
Remove the footer tag and just have it as text. (can center it using text-align: center; on the titleBackgroundFooter class